webapp/app/components/Main.test.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Main } from './Main'; // Undecorated version.
+import { keys } from 'lodash/fp';
+import { Main, mapDispatchToProps, mapStateToProps } from './Main'; // Undecorated version.
+import initialState from '../reducers/initialState';
 
 function setup(props) {
   return shallow(<Main {...props} />);
 }
 
 describe('<Main /> component', () => {
   it('renders itself', () => {
+    // Arrange
     const wrapper = setup({
       actions: {},
       usersActions: {}
     });
 
+    // Assert
     expect(wrapper.find('Switch')).toHaveLength(1);
     expect(wrapper.find('Route')).toHaveLength(3);
   });
+
+  describe('mapToProps functions', () => {
```

Comment:
`mapToProps` should be `mapStateToProps` function and a separate describe block for `mapDispatchToProps`